### ISSUE TITLE
mad mimi mailer wants all our mails to send out via madmimi

### ISSUE THREAD

**Issue by aka47** [redsquirrel/mad_mimi_mailer#4]
Hi,

with the update to the latest version, mad-mini wants to send out all emails via mad-mimi. Does not matter if they have mimi_ in their name or not. 

What are we doing wrong, since your commit tries exactly to fix this...
We downgraded for now to 0.0.9 and when i have time i will have a closer look.

**Comment by madmimi** [comment]
Were you seeing this issue on 0.1.2.1?

**Comment by aka47** [comment]
yes, 0.1.2.1. I did not try it in production, but our specs are failing because of it.

I another project, where every mail gets send out via mad-mimi, the problem was that it set the promotion not correct, because method_prefix was nil. (They use it still via inherintence not as mix-in.)

**Comment by mech** [comment]
I have similar problem. During the 0.0.8 version, I need to specify mimi_ prefix on my Mailer class and without the mimi_ prefix on my exact Promotion name, now I need to delete all my mimi_ prefix method. Is this the intended behavior?

# previously

def mimi_notify_candidate
end

# now, because the promotion name is "notify_candidate"

def notify_candidate
end

Can someone point me to where MadMimi has make such changes or some announcement of it :)

Thanks
